app/controllers/package_set.js

```diff
@@ -49,6 +49,15 @@ export default Controller.extend({
       return record.get("isSet") ? record.get("packages") : [record];
     }
   ),
+  packageUnavailableInSet: computed(
+    "allPackages.@each.availableQuantity",
+    function() {
+      let quantity = this.get("allPackages").any(
+        pkg => pkg.get("availableQuantity") == 0
+      );
+      return quantity ? true : false;
```

Comment:
`!!quantity` would be better.

Reply:
its a boolean function.
You can name it something with `isXYZ` kind of name. For example: `isPackageUnavailableInSet`

Further you can make it as
```javascript
isPackageUnavailableInSet: computed(
    "allPackages.@each.availableQuantity",
    () => this.get("allPackages").any(pkg => !pkg.get("availableQuantity"))
)
```